app/Filament/Resources/AchievementResource.php

```diff
@@ -73,94 +75,53 @@ public static function infolist(Schema $schema): Schema
-                                ]),
-
-                            Schemas\Components\Group::make()
-                                ->schema([
-                                    Infolists\Components\TextEntry::make('canonical_url')
-                                        ->label('Canonical URL')
-                                        ->url(fn (Achievement $record): string => $record->getCanonicalUrlAttribute()),
-
-                                    Infolists\Components\TextEntry::make('permalink')
-                                        ->url(fn (Achievement $record): string => $record->getPermalinkAttribute()),
-
-                                    Infolists\Components\TextEntry::make('activeMaintainer.user.display_name')
-                                        ->label('Current Maintainer')
-                                        ->placeholder(fn (Achievement $record) => $record->developer->display_name . ' (original developer)')
-                                        ->formatStateUsing(fn ($state, Achievement $record) => $state . ' (since ' .
-                                            ($record->activeMaintainer?->effective_from?->format('Y-m-d') ?? 'N/A') . ')')
-                                        ->extraAttributes(['class' => 'font-medium']),
-
-                                    Schemas\Components\Actions::make([
-                                        Actions\Action::make('setMaintainer')
-                                            ->label('Change Maintainer')
-                                            ->icon('heroicon-o-user')
-                                            ->schema(fn (Achievement $record) => static::buildMaintainerForm($record))
-                                            ->action(function (Achievement $record, array $data): void {
-                                                static::handleSetMaintainer($record, $data);
-
-                                                Notification::make()
-                                                    ->success()
-                                                    ->title('Success')
-                                                    ->body('Set achievement maintainer.')
-                                                    ->send();
-                                            }),
-                                    ])
-                                        ->visible(fn (): bool => $user->can('assignMaintainer', [Achievement::class])),
-                                ]),
-                        ]),
+                    Infolists\Components\ImageEntry::make('badge_url')
+                        ->label('Icon')
+                        ->size(config('media.icon.lg.width')),
+
+                    Infolists\Components\ImageEntry::make('badge_locked_url')
+                        ->label('Icon (Locked)')
+                        ->size(config('media.icon.lg.width')),
```

Comment:
These images are stored at 64x64, but displayed at 96x96, causing them to be a little fuzzy.

Reply:
Ah, my bad. I always get that mixed up with these. Updated to use `.md.width` in latest.